.github/workflows/e2e.yaml

```
@@ -31,3 +28,21 @@ jobs:
       - name: Debug failure
         if: failure()
         run: test/debug.sh
+  kubernetes-gpg:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: engineerd/setup-kind@v0.3.0
+      - name: Build container
+        run: test/build.sh
+      - name: Generate SSH key
+        run: test/ssh-keygen.sh
+      - name: Generate GPG key
+        run: test/gpg-keygen.sh
+      - name: Deploy and verify
+        run: test/deploy.sh deploy-gpg
+      - name: Generate known_hosts
+        run: test/ssh-keyscan.sh
+      - name: Debug failure
+        #if: failure()
```

```suggestion
        #if: failure()
```

Is this on purpose or a left-over from debugging?

---

Ah I forgot to remove the comment.